src/test/kotlin/study/CheckListTest.kt

```diff
@@ -0,0 +1,45 @@
+package study
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatThrownBy
+import org.junit.jupiter.api.Test
+import java.lang.IllegalArgumentException
+
+class CheckListTest {
+
+    @Test
+    fun check_list() {
+        val checkList = CheckList
+        val numList = listOf("1", "2", "3")
+        val symbolList = listOf("+", "-")
+        val testNumList = mutableListOf<Int>(1, 2, 3)
```

Comment:
타입이 명확하기때문에 Generic을 사용해 표기하지 않아도 괜찮을것 같아요!
다른 부분들은 다 빠져있는데 여기만 붙어있어서 코멘트 남겼습니다.

```suggestion
        val testNumList = mutableListOf(1, 2, 3)
```